lib/stdlib/src/math.erl

```diff
@@ -134,8 +134,31 @@ tan(_) ->
 tanh(_) ->
     erlang:nif_error(undef).
 
+-spec ceilf(X) -> float() when
+      X :: number().
+ceilf(_) ->
+    erlang:nif_error(undef).
+
+-spec floorf(X) -> float() when
+      X :: number().
+floorf(_) ->
+    erlang:nif_error(undef).
+
 %%% End of BIFs
 
+-spec ceil(X) -> integer() when
```

Comment:
Well it didn't in when I made the measurements, have you measured?

I did a loop of 10' millions calls to math:[int_]ceil(rand:normal()) with a set seed and keeping the last result in the loop, in spawned processes. 
Most of the time will be spent in rand:normal() but that overhead should be the same for both tests.